.env.example

```diff
@@ -0,0 +1,15 @@
+# Flag to switch to the production environment (redirect urls, gateway url, etc)
+NEXT_PUBLIC_IS_PRODUCTION=false
+
+# Latest supported safe version, used for upgrade prompts
+NEXT_PUBLIC_SAFE_VERSION=1.3.0
+
+# Access keys
+NEXT_PUBLIC_INFURA_TOKEN=
```

Comment:
AFAIK this is the only really necessary var, so maybe we should mark it as such. Otherwise it's overwhelming for new devs to find all these other keys.

Reply:
Do you know if fortmatic and portis wallets work without a key set? There's no fallback defined for these vars in the code:
> export const FORTMATIC_KEY = process.env.NEXT_PUBLIC_FORTMATIC_KEY || ''
export const PORTIS_KEY = process.env.NEXT_PUBLIC_PORTIS_KEY || ''

If not, we may want to mark them required as well 🤔

Reply:
They are not required to run the code locally assuming you're a normal person with metamask.